app/config/sculpin_site.yml

```diff
@@ -85,6 +85,9 @@ subspecies:
     rainbow:
         common: "Rainbow"
         latin: "arcus"
+    siteground:
+        common: "SiteGround"
+        latin: "SiteGround elephantus"
```

Comment:
This doesn't work as a subspecies name. Typically we use the name or something related, loosely translated into Latin. Since "SinceGround" isn't a real word, it's a little harder. Is there a good name we can translate?